.github/workflows/master_only.yml

```diff
@@ -81,7 +81,7 @@ jobs:
           --health-timeout 5s
           --health-retries 5
     steps:
-      - uses: actions/checkout@v5
+      - uses: actions/checkout@v3
```

Comment:
v4?

https://github.com/actions/checkout?tab=readme-ov-file#checkout-v4

Reply:
We have a couple dozen of em, probably need to get them all at once. Can we fix tests with this PR and I'll open another to update the whole lot from 3 --> 4?